### PR TITLE
feat(business): apply textexpand business styling

### PR DESCRIPTION
**Pull request by vlanz**
add business styling for textexpand component

### CODE REVIEW

File: projects/angular-showcase/src/app/business/business-examples/textexpand-showcase/textexpand-showcase.component.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'sbb-textexpand-showcase',
+  templateUrl: './textexpand-showcase.component.html',
+  styleUrls: ['./textexpand-showcase.component.scss']
```
Comment (jeripeierSBB) [inline projects/angular-showcase/src/app/business/business-examples/textexpand-showcase/textexpand-showcase.component.ts]: You can remove styleUrls and the corresponding css file, when the css file is empty anyway.

File: projects/angular-showcase/src/app/business/business-examples/textexpand-showcase/textexpand-showcase.component.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<div class="container-fluid">
+  <div class="row">
+    <div class="col-12">
+      <h4 class="sbbsc-block">Basic Example</h4>
+      <sbb-textexpand>
```
Comment (jeripeierSBB) [inline projects/angular-showcase/src/app/business/business-examples/textexpand-showcase/textexpand-showcase.component.html]: You can remove everyhing around the <sbb-textexpand> tag to achieve a minimal example. If you like, improve it also in public. Thanks :)

**Comment by jeripeierSBB** [comment]
@frenggi Can you check the styles at https://angular-staging.app.sbb.ch/351/business/components/textexpand ? Thanks :)

**Comment by Frenggi** [comment]
It looks almost fine to me. But I miss the hover-effect for the link "mehr anzeigen" / "weniger anzeigen".

**Comment by vlanz** [comment]
> It looks almost fine to me. But I miss the hover-effect for the link "mehr anzeigen" / "weniger anzeigen".

@Frenggi I didn't see any specifications for hover behaviour at this [link](https://digital.sbb.ch/en/webapps/components/textexpand). What would you expect?

**Comment by Frenggi** [comment]
> > It looks almost fine to me. But I miss the hover-effect for the link "mehr anzeigen" / "weniger anzeigen".
> 
> @Frenggi I didn't see any specifications for hover behaviour at this [link](https://digital.sbb.ch/en/webapps/components/textexpand). What would you expect?

I expect the same behaviour for a normal link: https://digital.sbb.ch/de/webapps/components/link

**Pull request merged**